### PR TITLE
WINDUP-836 changed comments as they influenced wrongly loading of sty…

### PR DESCRIPTION
**Pull request by mareknovotny**
…les for .label and .label-info
tag with .label and .label-info looks like in the following screenshot https://www.dropbox.com/s/1uk6xqj54jts1fb/Screenshot-labels.png

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1249/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/623/

**Comment by mareknovotny** [comment]
if we don't merge this PR we should rolled back https://github.com/windup/windup/commit/b8da773e4fb98f8a238b5fb7b52845854783cf43 commit as that breaks the css styles for .label and .label-info

File: reporting/impl/src/main/resources/reports/resources/css/windup.css [hunk 1]
```diff
@@ -116,7 +116,7 @@ dl:last-child {
     margin: 2px;
     font-weight: bold;
 }
-//overwriting bootstrap style
+/*overwriting bootstrap style*/
 .label-info {
     background-color: #696969; //this is dark grey instead of blueish from bootstrap's label-info
```
Comment (OndraZizka) [inline reporting/impl/src/main/resources/reports/resources/css/windup.css]: Still wrong :)

**Comment by OndraZizka** [comment]
Could you please make a screenshot and link?

File: reporting/impl/src/main/resources/reports/resources/css/windup.css (hunk 1 above)
Comment (mareknovotny) [inline reporting/impl/src/main/resources/reports/resources/css/windup.css]: gotcha, even this line doesn't result in error ;) I will fix it

**Comment by mareknovotny** [comment]
@OndraZizka see the firs comment with link to screenshot, but as favour for you pasting again here 
https://www.dropbox.com/s/1uk6xqj54jts1fb/Screenshot-labels.png

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/629/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1255/

**Pull request merged**